WebRoot/js/zimbraMail/calendar/view/ZmExternalCalendarDialog.js

```diff
@@ -27,20 +27,22 @@ ZmExternalCalendarDialog = function(params) {
     var cancel = new DwtDialog_ButtonDescriptor(ZmExternalCalendarDialog.SHARE_CANCEL_BUTTON, ZmMsg.cancel, DwtDialog.ALIGN_RIGHT);
     var parent = params.parent || appCtxt.getShell();
     this._controller = params.controller;
-    ZmDialog.call(this, {parent:parent, className:"ZmExternalCalendarDialog", standardButtons:[DwtDialog.NO_BUTTONS], extraButtons: [back, next, cancel], id:'ADD_EXTERNAL_CAL_DIALOG'});
+    ZmDialog.call(this, {parent:parent, className:"ZmExternalCalendarDialog", standardButtons:[DwtDialog.NO_BUTTONS], extraButtons: [back, cancel, next], id:'ADD_EXTERNAL_CAL_DIALOG'});
 
 	this.setButtonListener(ZmExternalCalendarDialog.BACK_BUTTON, new AjxListener(this, this._backButtonListener));
 	this.setButtonListener(ZmExternalCalendarDialog.NEXT_BUTTON, new AjxListener(this, this._nextButtonListener));
 	this.setButtonListener(ZmExternalCalendarDialog.SHARE_CANCEL_BUTTON, new AjxListener(this, this._cancelButtonListener));
 
+	this.getButton(ZmExternalCalendarDialog.NEXT_BUTTON).addClassName('ZPrimaryButton');
```

Comment:
Minor: alignment fro line 32 to 36. It was already misaligned. can we correct it?

Reply:
fixed in https://github.com/Zimbra/zm-web-client/pull/140/commits/c476e679237602e6d9249ab741066eb924ddcdd0